### PR TITLE
Update docs to reflect v1 feature set

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation
#### What this PR does / why we need it:
The profile and node validation is now part of the mentioned feature
set.
#### Which issue(s) this PR fixes:
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: installation-usage.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The feature scope of the seccomp-operator is right now limited to:
 
 - Enabling `ConfigMap`s to store seccomp profiles.
 - Synchronizing seccomp profiles across all worker nodes.
+- Validate if a node supports seccomp and do not synchronize if not.
+- Validate if a profile is syntactical correct and do not synchronize if not.
```
Comment (pjbgf) [inline installation-usage.md:10]: nit: we could keep all sentences on the same tense, by removing the present continuous from the first two:

```suggestion
- Enable `ConfigMap`s to store seccomp profiles.
- Synchronize seccomp profiles across all worker nodes.
- Validate if a node supports seccomp and do not synchronize if not.
- Validate if a profile is syntactically correct and do not synchronize if not.
```

Comment (saschagrunert) [reply installation-usage.md]: Adapted as suggested :+1:

**Comment by hasheddan** [comment]
Added hold to block my lgtm, looks like changes have been addressed to feel free to remove!

**Comment by saschagrunert** [comment]
Yep 
/hold cancel

**Comment by saschagrunert** [comment]
@pjbgf PTAL again

**Pull request merged**